tests/e2e/features/dicomViewer.feature

```diff
@@ -5,8 +5,8 @@ Feature: Preview dicom image
 
 
   Scenario: user previews dicom image
-    Given the user "Admin" has logged in
-    And the user has uploaded the dicom file "MRBRAIN.dcm"
+    Given the user has uploaded the dicom file "MRBRAIN.dcm"
```

Comment:
I think you can use passive verbs structure in the Given statement, then you don't need to specify who has uploaded the file and how this was done (manually or using the API)

Example: 
Given the dicom file XYZ has been uploaded